Recipely/Recipely/NetworkService/RecipeDetailResponseDTO.swift

```diff
@@ -0,0 +1,80 @@
+// RecipeDetailResponseDTO.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+
+/// RecipeDetailResponseDTO
```

Comment:
```suggestion
/// RecipeDetailResponseDTO
```
дока не соответствует действительности